<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PridatKnihu</class>
 <widget class="QDialog" name="PridatKnihu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Přidat knihu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PridatKnihu {
background: rgb(204,229,250);
}
#lineEditAutor1{
background: rgb(235,235,235);
}
#lineEditAutor2{
background: rgb(235,235,235);
}
#lineEditCena{
background: rgb(235,235,235);
}
#lineEditMisto{
background: rgb(235,235,235);
}
#lineEditNazevKnihy{
background: rgb(235,235,235);
}
#lineEditTematika{
background: rgb(235,235,235);
}
#spinBoxEvidencniCislo{
background: rgb(235,235,235);
}
#spinBoxRok{
background: rgb(235,235,235);
}
#spinBoxStrana{
background: rgb(235,235,235);
}


</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_7{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Tématika</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Evdidenční číslo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_2{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Název knihy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_4{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Autor2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_3{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Autor 1</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_9{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Počet stran</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_5{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Nakladatelství</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_8{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Cena</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#label_6{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Rok vydání</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="radioButtonKniha">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#radioButtonKniha{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Kniha</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QRadioButton" name="radioButtonCD">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#radioButtonCD{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>CD</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBoxEvidencniCislo">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditNazevKnihy">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditAutor1">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditAutor2">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditMisto">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBoxRok">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEditTematika">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEditCena">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spinBoxStrana">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>25</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PridatKnihu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PridatKnihu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
